src/main/resources/assets/mmorpg/lang/zh_cn.json

```diff
@@ -301,176 +275,137 @@
 	"mmorpg.affix.thirst_of_frost": "嗜寒的",
 	"mmorpg.affix.thirst_of_lightning": "嗜雷的",
 	"mmorpg.affix.thorny": "棘刺的",
-	"mmorpg.affix.thunder_heretic": "雷电异教徒的",
-	"mmorpg.affix.thunder_imbued": "雷电浸透的",
-	"mmorpg.affix.thunder_shield": "雷电保护的",
+	"mmorpg.affix.thunder_heretic": "雷电异端的",
+	"mmorpg.affix.thunder_imbued": "注雷的",
+	"mmorpg.affix.thunder_shield": "雷电护卫的",
 	"mmorpg.affix.thundering": "雷鸣的",
-	"mmorpg.affix.touch_of_magic": "法神的",
+	"mmorpg.affix.touch_of_magic": "魔法之触的",
 	"mmorpg.affix.tough": "坚韧的",
 	"mmorpg.affix.treasure_seeker": "寻宝者的",
-	"mmorpg.affix.unending_field": "无尽领域的",
-	"mmorpg.affix.water_heretic": "冰霜异教徒的",
-	"mmorpg.affix.water_imbued": "冰霜浸透的",
-	"mmorpg.affix.water_shield": "水系保护的",
+	"mmorpg.affix.unending_field": "不竭领域的",
+	"mmorpg.affix.water_heretic": "冰霜异端的",
+	"mmorpg.affix.water_imbued": "注霜的",
+	"mmorpg.affix.water_shield": "冰霜护盾",
 	"mmorpg.affix.wise": "智慧的",
-	
-"_comment": " [CATEGORY]: AFFIXES - NAMES",
-
-
-"_comment": " [CATEGORY]: ITEM SETS - NAMES",
-
-	"mmorpg.set.archemage_vestments": "法师的长袍",
+	"mmorpg.set.archemage_vestments": "法师长袍",
 	"mmorpg.set.archmage_armor": "大法师盔甲",
 	"mmorpg.set.armor_of_the_elements": "元素盔甲",
-	"mmorpg.set.artifact_armor": "人造盔甲",
+	"mmorpg.set.artifact_armor": "精装盔甲",
 	"mmorpg.set.ascension_of_fire": "火焰提升",
 	"mmorpg.set.ascension_of_nature": "自然提升",
 	"mmorpg.set.ascension_of_thunder": "雷电提升",
-	"mmorpg.set.ascension_of_water": "水系提升",
-	"mmorpg.set.barbarian_armor": "狂战士的盔甲",
+	"mmorpg.set.ascension_of_water": "寒霜提升",
+	"mmorpg.set.barbarian_armor": "野蛮人盔甲",
 	"mmorpg.set.elemental_essence": "元素精华",
-	"mmorpg.set.eliminator": "消除",
-	"mmorpg.set.flaming_devil": "燃烧的恶魔",
+	"mmorpg.set.eliminator": "无情杀手",
+	"mmorpg.set.flaming_devil": "燃焰恶魔",
 	"mmorpg.set.forest_guardian": "森林守护者",
-	"mmorpg.set.godkings_plate": "不死之王",
+	"mmorpg.set.godkings_plate": "神王护板",
 	"mmorpg.set.ice_lord": "冰之领主",
 	"mmorpg.set.limitless": "无限",
 	"mmorpg.set.mystical_ornaments": "神秘饰品",
-	"mmorpg.set.ranger_armor": "游侠的盔甲",
+	"mmorpg.set.ranger_armor": "游侠盔甲",
 	"mmorpg.set.rockman_chains": "洛克人的锁链",
-	"mmorpg.set.royal_thief_adornments": "王室盗窃的饰品",
-	"mmorpg.set.scholars_robes": "学者的长袍",
+	"mmorpg.set.royal_thief_adornments": "王室窃贼的饰品",
+	"mmorpg.set.scholars_robes": "学者长袍",
 	"mmorpg.set.seers_guidance": "预言者的指引",
-	"mmorpg.set.set_rings_of_impossibility": "不可能的戒指",
+	"mmorpg.set.set_rings_of_impossibility": "不可能之戒",
 	"mmorpg.set.spirit_of_the_arcane": "奥术之魂",
-	"mmorpg.set.the_ascended": "攀登者",
+	"mmorpg.set.the_ascended": "飞升者",
 	"mmorpg.set.the_protector": "守护者",
 	"mmorpg.set.treasure_finder": "寻宝者",
 	"mmorpg.set.tree_of_life": "生命之树",
-	"mmorpg.set.will_of_lightning": "雷电意志",
+	"mmorpg.set.will_of_lightning": "闪电意志",
 	"mmorpg.set.wisdom_of_the_elders": "长者的智慧",
```

Comment:
This part I reworked the Spell, affix and set translation. The old translation have consistency problem in Spell name, and not that accurate and formal. I change almost all translations of Spell name, and keep it all consisted because of the CAT. The problem of affix and set is same as the Spell name.